tests/systemtests/rest_cli.go

```
@@ -66,14 +66,23 @@
 
 	for key, value := range headers {
 		req.Header.Set(key, value)
 	}
 
-	httpClient := &http.Client{}
+
+	println(`req : `, req.URL.String())
+	httpClient := &http.Client{
+		CheckRedirect: func(req *http.Request, via []*http.Request) error {
+			println(`redirect : `, req.URL.String())
+			return nil 
```

golangci-lint (1.61.0)

File is not `gci`-ed with --skip-generated -s standard -s default -s prefix(cosmossdk.io) -s prefix(github.com/cosmos/cosmos-sdk) --custom-order

(gci)